cardano-node/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -599,7 +601,7 @@ instance (Condense (HeaderHash blk), ProtocolLedgerView blk)
   toObject MinimalVerbosity p = toObject NormalVerbosity p
   toObject verb p =
     mkObject [ "kind" .= String "Tip"
-             , "tip" .= showTip verb p ]
+             , "tip" .= showPoint verb p ]
```

Comment:
@dcoutts: I know this isn't your code, but this doesn't seem right. Shouldn't the `kind` be `"Point"`? Since this is a `Point blk`, this wouldn't necessarily be the tip.

Reply:
Yes I think you're right, it should.

We may want a separate `ToObject` instance for the type `Tip`.

Reply:
Opened an issue so it can be addressed later: https://github.com/input-output-hk/cardano-node/issues/570